hedyorg/hedy
# 🪲 Fix list printing

Fixes #5714
Address printing lists.

**How to test**
Go to level 16 and execute the following program. Ensure that there are no errors and the list is printed.
```
mylist = [1, 2, 3]
print mylist
```
